app/templates/DisplayPageLayout.twig

```diff
@@ -56,7 +56,7 @@
     <!-- Header -->
     <header id="header">
         <div class="container clearfix">
-            {$ header | raw $}
+            {% include 'header' %}
```

Comment:
No. When the `header` template is not found, there is an error message. But that's a good thing since that will point to a mismatch between the includes and the arguments that were passed to `renderArgumentsAsTemplates`. The unit tests will catch this.